ui/pages/add-collectible/add-collectible.js

```diff
@@ -95,8 +75,9 @@ export default function AddCollectible() {
     }
     dispatch(setNewCollectibleAddedMessage('success'));
 
-    const tokenStandard = await getCollectibleStandard(
+    const tokenDetails = await getTokenStandardAndDetails(
       address,
+      selectedAddress,
```

Comment:
We actually don't need this here since it is not used for non-ERC20 tokens. Its a weird function signature that we will change (to accept an object) but for now you can and should put the second arg as `null` and no longer need to get the selectedAddress from state.